smoke-tests/spec/external_dns_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "spec_helper"
+
+describe "external DNS" do
+  # let!(zone) { nil }
+
+  context "when zone matches ingress domain" do
+    let(:domain) { "child.parent.service.justice.gov.uk" }
+    let(:namespace) { "child-parent" }
+    let(:ingress_domain) { domain }
+    let(:parent_zone_id) { "ZQVC43X15AWL9" }
+    zone = nil
+
+    # Create a new zone per test
+    before do
+      zone = create_zone(domain)
```

Comment:
Since you're not using `before(:all)` I think you might be able to do this with `let` statements. Have you tried that?

Reply:
☝️